server/handlers/user_handler.coffee

```diff
@@ -497,8 +497,6 @@ UserHandler = class UserHandler extends Handler
       photoURL = document?.get('photoURL')
       if photoURL
         photoURL = "/file/#{photoURL}"
```

Comment:
I was going to leave it in in case we start letting users choose avatars from a list, then this could would be useful again. This will also let people continue to use their existing avatars until we actually go through and delete them after we have a backup.